pkg/controller/common/helpers.go

```diff
@@ -278,6 +280,28 @@ func convertIgnition32to31(ign3config ign3types.Config) (ign3_1types.Config, err
 	return converted31, nil
 }
 
+// AddIgnitionKernelArguments adds the given kernel arguments to the Ignition configs
+// kernelArguments.shouldExist field
+func AddIgnitionKernelArguments(inRawExtIgn *runtime.RawExtension, kargs []string) (*runtime.RawExtension, error) {
+	outRawExt := &runtime.RawExtension{}
+	ignCfg, rptV3, errV3 := ign3_3.ParseCompatibleVersion(inRawExtIgn.Raw)
+	if errV3 != nil || rptV3.IsFatal() {
+		return outRawExt, errors.Errorf("parsing Ignition config failed with error: %v\nReport: %v", errV3, rptV3)
+	}
+
+	for _, karg := range kargs {
+		ignCfg.KernelArguments.ShouldExist = append(ignCfg.KernelArguments.ShouldExist, ign3_3types.KernelArgument(karg))
+	}
+
+	outIgn, err := json.Marshal(ignCfg)
+	if err != nil {
+		return outRawExt, errors.Errorf("failed to marshal converted config: %v", err)
```

Comment:
this message is from here: https://github.com/openshift/machine-config-operator/blob/8bd0dd547059e9a5cbb43d2f58b40e6bece6c72f/pkg/controller/common/helpers.go#L177 
but i dont think converting is applicable in this case?

Reply:
How so?

Reply:
when i read it yesterday it didnt seem like this was a converted config? (correct me if im wrong)

Reply:
> @darkmuggle sorry I think I left my comments prematurely - noticing there's a hold 😬

pfft :) happy to have early feedback